libnmstate/netapplier.py

```diff
@@ -63,9 +63,20 @@ def apply(desired_state, verify_change=True, commit=True, rollback_timeout=60):
     validator.validate_dns(desired_state)
     validator.validate_vxlan(desired_state)
 
+    not_desired_state_vlans = _prepare_vlan_ifaces_for_mtu_change(
+        desired_state
+    )
     checkpoint = _apply_ifaces_state(
         state.State(desired_state), verify_change, commit, rollback_timeout
     )
+    if not_desired_state_vlans:
+        _apply_ifaces_state(
+            state.State({schema.Interface.KEY: not_desired_state_vlans}),
+            False,
```

Comment:
this `False` value as is is hard to understand what it stands for. As such, I'd use `verify_change=False` instead. 

Also, why don't we check the state after the transaction commit ?

Reply:
Oops, we are using verify_change variable value now. Thanks!